src/picterra/client.py

```diff
@@ -386,10 +388,36 @@ def download_result_to_file(self, operation_id, filename):
             r.raise_for_status()
             with open(filename, 'wb') as f:
                 logger.debug('Trying to save result to file %s..' % filename)
-                for chunk in r.iter_content(chunk_size=8192):
+                for chunk in r.iter_content(chunk_size=CHUNK_SIZE_BYTES):
                     if chunk:  # filter out keep-alive new chunks
                         f.write(chunk)
 
+    def download_url_to_file(self, operation_id, filename):
+        """
+        Downloads the results URL to a local GeoJSON file
+
+        Args:
+            result_id (str): The id of the result to download
+            filename (str): The local filename where to save the results
+        """
+        data = self.get_result_url(operation_id)
+        with open(filename, 'w') as f:
+            f.write(data)
+
+    def get_result_url(self, operation_id: str) -> str:
+        """
+        Get the URL  of a set of results
+
+        This a **beta** function, subject to change.
+
+        Args:
+            result_id (str): The id of the result
+        """
+        resp = self.sess.get(
+            self._api_url('operations/%s/' % operation_id),
+        )
+        return resp.json()['results']['url']
```

Comment:
We should keep in mind that this might raise `KeyError` for some operations, as well as incomplete operations. Not something to fix now I think, but in the future we might want to handle these cases by raising an `APIError` with a nicer error message.

Reply:
Checked (added a comment)